examples/3d_drawing/plot_3d_animation_basic.py

```diff
@@ -0,0 +1,60 @@
+"""
+===============
+Basic Animation
+===============
+
+NetworkX supports several 3D layout functions for visualization.
+These layouts can be combined with :external+matplotlib:doc:`api/animation_api`
+for 3D graph visualization.
+
+This example shows a basic animation incrementing the camera view.
+"""
+import numpy as np
+import networkx as nx
+import matplotlib.pyplot as plt
+from matplotlib import animation
+
+###############################################################################
+# Define a graph to plot.
+# -----------------------
+#
+# Pick a graph for visualization in 3D, such as one of the Platonic solids
+
+G = nx.dodecahedral_graph()
+pos = nx.spring_layout(G, dim=3, seed=2734)
+nodes = np.array([pos[v] for v in G])
+edges = np.array([(pos[u], pos[v]) for u, v in G.edges()])
+
+###############################################################################
+# Rotating 3D graph animation.
+# ----------------------------
+#
+# In this example, a frame update is only a rotation of a given 3D graph.
+
+
+def init():
```

Comment:
do not you need here
```
ax.clear()
```
?

Reply:
It's necessary to remove the initial point that is added to represent the current node. Without this, one of the vertices of the dodecahedron retains a permanent red square.